os/kernel/task/task_activate.c

```diff
@@ -84,6 +93,40 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#ifndef CONFIG_DISABLE_SIGNALS
+void default_kill_handler(int signo, siginfo_t *data)
```

Comment:
`default_kill_handler` does not represent what it works.
Let's rename it like stop or terminate thread.